mobile/cmp/input/Select.scss

```diff
@@ -81,3 +81,37 @@
     }
   }
 }
+
+//---------------
+// Fullscreen
+//---------------
+#xh-select-input-fullscreen-portal {
+  z-index: 22;
+
+  .xh-select {
+    margin: var(--xh-pad-px);
+
+    &__fullscreen-wrapper {
+      position: absolute !important;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
```

Comment:
Is it standard to specify right and bottom like this (in addition to top/left) for an absolutely positioned element?

Reply:
It's another way of making an absolute positioned element fill its container. We could also give it `top: 0; left: 0; width: 100%; height: 100%;`, but I think that might have some unexpected behaviour in some circumstances (i.e. container is floating).

https://rajatamil.medium.com/css-make-a-div-full-screen-402f9a935d15

We're using it in many other places throughout the code (CodeInput, TreeMap, Mask, etc)

Reply:
Nice, thanks for the info - will (attempt to) remember it for when I need to do the same!